profitbricks/src/test/java/org/jclouds/profitbricks/http/parser/loadbalancer/LoadBalancerListResponseHandlerTest.java

```diff
@@ -0,0 +1,75 @@
+import org.jclouds.profitbricks.domain.Server;
+import org.jclouds.profitbricks.http.parser.BaseResponseHandlerTest;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import org.testng.annotations.Test;
+
+@Test(groups = "unit", testName = "LoadbalancerListResponseHandlerTest")
+public class LoadBalancerListResponseHandlerTest extends BaseResponseHandlerTest<List<LoadBalancer>> {
+
+   @Override
+   protected ParseSax<List<LoadBalancer>> createParser() {
+      return factory.create(injector.getInstance(LoadBalancerListResponseHandler.class));
+   }
+
+   protected DateCodecFactory createDateParser() {
+      return injector.getInstance(DateCodecFactory.class);
+   }
+
+   @Test
+   public void testParseResponseFromGetAllLoadbalancer() {
+      ParseSax<List<LoadBalancer>> parser = createParser();
+
+      List<LoadBalancer> actual = parser.parse(payloadFromResource("/loadbalancer/loadbalancers.xml"));
+      assertNotNull(actual, "Parsed content returned null");
+
+      DateCodec dateParser = createDateParser().iso8601();
+
+      List<LoadBalancer> expected = ImmutableList.<LoadBalancer>of(LoadBalancer.builder().id("aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee").loadBalancerAlgorithm(LoadbalancerAlgorithm.ROUND_ROBIN).name("load-1234567890-name")
+              .dataCenterId("datacenter-id").dataCenterVersion("datacenter-version").internetAccess(true).ip("ip").lanId("lan-id").state(ProvisioningState.AVAILABLE).creationTime(dateParser.toDate("2014-12-04T07:09:23.138Z")).lastModificationTime(dateParser.toDate("2014-12-04T07:09:23.138Z"))
+              .firewalls(ImmutableList.<Firewall>of(
+                              Firewall.builder().id("firewall-id").nicId("nic-id").active(false).state(ProvisioningState.AVAILABLE).build()
+                      ))
+              .balancedServers(ImmutableList.<Server>of(
+                              Server.builder().activate(true).balancedNicId("balanced-nic-id").id("server-id").name("server-name").build()
+                      )).build(),
+              LoadBalancer.builder().id("qqqqqqqq-wwww-rrrr-tttt-yyyyyyyyyyyy").loadBalancerAlgorithm(LoadbalancerAlgorithm.ROUND_ROBIN).name("load-balancer-name")
+              .dataCenterId("datacenter-id").dataCenterVersion("datacenter-version").internetAccess(false).ip("ip").lanId("lan-id").state(ProvisioningState.AVAILABLE).creationTime(dateParser.toDate("2014-12-04T07:09:23.138Z")).lastModificationTime(dateParser.toDate("2014-12-04T07:09:23.138Z"))
+              .firewalls(ImmutableList.<Firewall>of(
+                              Firewall.builder().id("firewall-id").nicId("nic-id").active(false).state(ProvisioningState.AVAILABLE).build()
+                      ))
+              .balancedServers(ImmutableList.<Server>of(
+                              Server.builder().activate(false).balancedNicId("balanced-nic-id").id("server-id").name("server-name").build()
+                      ))
+              .build()
```

Comment:
Could you format the builder such that each property set is in it's own line? See one [from Server](https://github.com/devcsrj/jclouds-labs/blob/master/profitbricks/src/test/java/org/jclouds/profitbricks/http/parser/server/ServerInfoResponseHandlerTest.java#L59-L108) for example (Or your `LoadBalancerResponseHandlerTest`). This is so we could quickly browse to the code on cases the test fails (readability)